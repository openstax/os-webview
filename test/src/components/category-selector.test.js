import CategorySelector from '~/components/category-selector/category-selector';

describe('CategorySelector', () => {
    const p = new CategorySelector();

    it('loads the categories', () => {
        return CategorySelector.loaded.then(() => {
            expect(CategorySelector.categories).toMatchSnapshot();
        });
    });
});
