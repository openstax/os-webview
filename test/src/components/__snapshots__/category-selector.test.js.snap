// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategorySelector loads the categories 1`] = `
Array [
  Object {
    "cms": "",
    "html": "View All",
    "value": "view-all",
  },
  Object {
    "cms": "Math",
    "html": "Math",
    "title": "Math SEO Title",
    "value": "math",
  },
  Object {
    "cms": "Science",
    "html": "Science",
    "title": "Science SEO Title",
    "value": "science",
  },
  Object {
    "cms": "Humanities",
    "html": "Humanities",
    "title": "Humanities SEO Title",
    "value": "humanities",
  },
  Object {
    "cms": "Social Sciences",
    "html": "Social Sciences",
    "title": "Social Sciences SEO Title",
    "value": "social-sciences",
  },
  Object {
    "cms": "Business",
    "html": "Business",
    "title": "Business SEO Title",
    "value": "business",
  },
]
`;
